ci-operator/step-registry/azure/provision/customer-managed-key-no-rbac/azure-provision-customer-managed-key-no-rbac-ref.metadata.json

```diff
@@ -0,0 +1,12 @@
+{
+	"path": "azure/provision/customer-managed-key-no-rbac/azure-provision-customer-managed-key-no-rbac-ref.yaml",
```

Comment:
compared with the existing `azure-provision-customer-managed-key` step, that step provisioned vault key, and used for `platform.auzre.customerManagedKey`, so we name like that, for this step, I suggest the name like `azure/provision/vault-key`

Reply:
Updated.